examples/nemo_ilql_sentiments.py

```diff
@@ -0,0 +1,49 @@
+import os
+from typing import Dict, List
+
+import yaml
+from datasets import load_dataset
+from transformers import pipeline
+
+import trlx
+from trlx.data.configs import TRLConfig
+
+
+def get_positive_score(scores):
+    "Extract value associated with a positive sentiment from pipeline's output"
+    return dict(map(lambda x: tuple(x.values()), scores))["POSITIVE"]
+
+
+default_config = yaml.safe_load(
+    open(os.path.dirname(__file__) + "/../configs/nemo_ilql_config.yml")
+)
+
+
+def main(hparams={}):
+    config = TRLConfig.update(default_config, hparams)
+
+    sentiment_fn = pipeline(
+        "sentiment-analysis",
+        "lvwerra/distilbert-imdb",
+        top_k=2,
+        truncation=True,
+        batch_size=256,
+        device=-1 if int(os.environ.get("LOCAL_RANK", 0)) == 0 else -1,
```

Comment:
I just tried again with 0 but it has to be CPU only for now, otherwise it seems to cause an OOM. I think it should be possible to have the reward function use GPU if its allocated on the right GPUs in the future though